src/libs/focusAndUpdateMultilineInputRange.ts

```diff
@@ -6,16 +7,14 @@
  * focus. This provides a better user experience in cases where the text in the field has to be edited. The auto-
  * scroll behaviour works on all platforms except iOS native.
  * See https://github.com/Expensify/App/issues/20836 for more details.
- *
- * @param {Object} input the input element
  */
-export default function focusAndUpdateMultilineInputRange(input) {
+export default function focusAndUpdateMultilineInputRange(input: TextInput | HTMLTextAreaElement) {
     if (!input) {
         return;
     }
 
     input.focus();
-    if (input.value && input.setSelectionRange) {
+    if ('setSelectionRange' in input && input.value) {
```

Comment:
Can we switch this to `_.has()` instead? As far as I can tell we don't use the `in` keyword in our codebase

Reply:
Hey! FYI I took over this PR as Eugene is out of office. Using `underscore` is forbidden in TypeScript files, afaik we are using `in` in some places e.g. `useDraggableInPortal.ts`. Let me know what you think!

Reply:
Gotcha. On second look, it does look like we are starting to use `in` in typescript